packages/app/src/app/overmind/namespaces/preferences/internalActions.ts

```diff
@@ -0,0 +1,36 @@
+import { Action } from 'app/overmind';
+import { SandboxOptions } from '@codesandbox/common/lib/url';
+
+export const updatePreferencesFromSandboxOptions: Action<SandboxOptions> = (
+  { state },
+  sandboxOptions
+) => {
+  state.preferences.showPreview =
+    sandboxOptions.isPreviewScreen || sandboxOptions.isSplitScreen;
+
+  state.preferences.showEditor =
+    sandboxOptions.isEditorScreen || sandboxOptions.isSplitScreen;
+
+  if (sandboxOptions.initialPath)
```

Comment:
@christianalfoni I think it's better to put curly braces around each `if`-block to make it less error-prone.

I know this will make this file a lot bigger, but (especially) for new(er) developers this will make the code a bit clearer.